admin/partials/cover_art.php

```diff
@@ -8,7 +8,7 @@
 <p class="description">
 	<?php printf(
 		wp_kses(
-			__( 'You can set one or more <a href="%s">cover art</a> images below.', 'apple-news' ),
+			__( 'You can set one or more <a href="%s">cover art</a> images below. Only one image is required in order to enable cover art functionality.', 'apple-news' ),
```

Comment:
I think this needs to explain a little better what Apple will do with autocropping. Maybe one more sentence?